src/components/AutocompleteTagBuilder/AutocompleteTagBuilder.types.part.tsx

```diff
@@ -0,0 +1,52 @@
+import { LabeledInputProps, InputChangeEvent } from '../../common';
+
+export interface AutocompleteTagBuilderAutosuggestItem {
+  key: string;
+  content?: React.ReactChild;
+}
+
+export interface AutocompleteTagBuilderProps<T> extends LabeledInputProps<Array<T>> {
```

Comment:
please introduce a new type with common props for `Autocomplete` and `AutocompleteTagBuilder` and share it between their props